collector/zoneinfo_linux.go

```diff
@@ -0,0 +1,227 @@
+			[]string{"node", "zone"}, nil),
+		"NrActiveFile": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_active_file"),
+			"Number of active pages with file-backing",
+			[]string{"node", "zone"}, nil),
+		"NrInactiveFile": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_inactive_file"),
+			"Number of inactive pages with file-backing",
+			[]string{"node", "zone"}, nil),
+		"NrIsolatedFile": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_isolated_file"),
+			"Temporary isolated pages from file lru",
+			[]string{"node", "zone"}, nil),
+		"NrFilePages": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_file_pages"),
+			"Number of file pages",
+			[]string{"node", "zone"}, nil),
+		"NrSlabReclaimable": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_slab_reclaimable"),
+			"Number of reclaimable slab pages",
+			[]string{"node", "zone"}, nil),
+		"NrSlabUnreclaimable": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_slab_unreclaimable"),
+			"Number of unreclaimable slab pages",
+			[]string{"node", "zone"}, nil),
+		"NrMlockStack": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_mlock_stack"),
+			"mlock()ed pages found and moved off LRU",
+			[]string{"node", "zone"}, nil),
+		"NrKernelStack": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_kernel_stack"),
+			"Amount of memory allocated to kernel stacks",
+			[]string{"node", "zone"}, nil),
+		"NrMapped": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_mapped"),
+			"Number of mapped paged",
+			[]string{"node", "zone"}, nil),
+		"NrDirty": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_dirty"),
+			"Number of dirty pages",
+			[]string{"node", "zone"}, nil),
+		"NrWriteback": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_writeback"),
+			"Number of writeback pages",
+			[]string{"node", "zone"}, nil),
+		"NrUnevictable": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_unevictable"),
+			"Number of unevictable pages",
+			[]string{"node", "zone"}, nil),
+		"NrShmem": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_shmem"),
+			"Number of shmem pages (included tmpfs/GEM pages)",
+			[]string{"node", "zone"}, nil),
+		"NrDirtied": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_dirtied"),
+			"Page dirtyings since bootup",
+			[]string{"node", "zone"}, nil),
+		"NrWritten": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "nr_written"),
+			"Page writings since bootup",
+			[]string{"node", "zone"}, nil),
+		"NumaHit": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, zoneinfoSubsystem, "numa_hit"),
```

Comment:
Are these hit/miss values counters? If they're counters, they should end in `_total` and be exported as counters.